src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -84,6 +84,18 @@ public function testGuessRequired($constraint, $guess)
         $this->assertEquals($guess, $this->guesser->guessRequired(self::TEST_CLASS, self::TEST_PROPERTY));
     }
 
+    /**
+     * @group legacy
+     */
+    public function testLegacyGuessRequired()
+    {
+      if (PHP_VERSION_ID >= 70000) {
+        $this->markTestSkipped('PHP is version 7 or higher.');
```

Comment:
Imho the message should provide the reason why this is kipped (something like "Cannot use a class called True on PHP 7 or higher").